static/js/common/upload-addon.js

```diff
@@ -130,6 +130,10 @@
                 $('#id_is_manual_review').prop('checked', false);
 
                 /* Don't allow submitting */
+                // The create theme wizard button is actually a link,
+                // so it's pointless to set the disabled property on it,
+                // instead add the special "concealed" class.
+                $('.addon-create-theme-section .button').addClass('concealed');
```

Comment:
Does this actually stop the link/button from working?  Or just makes it look disabled?

Reply:
It prevents you from clicking on it though CSS `pointer-events: none;`. It's not perfect, but properly disabling links is harder than it looks... Since this is how we do it for the rest of the site I figured it was enough.